tests/postgres_tests/test_functions.py

```diff
@@ -26,3 +27,20 @@ def test_transaction_now(self):
 
         self.assertIsInstance(m1.when, datetime)
         self.assertEqual(m1.when, m2.when)
+
+
+class TestRandomUUID(PostgreSQLTestCase):
+
+    def test_random_uuid(self):
+        m1 = UUIDTestModel.objects.create()
+        m2 = UUIDTestModel.objects.create()
+
+        UUIDTestModel.objects.filter(id=m1.id).update(uuid=RandomUUID())
+        sleep(0.1)
+        UUIDTestModel.objects.filter(id=m2.id).update(uuid=RandomUUID())
```

Comment:
This test would be better if we updated the whole model with the expression, like in the PR